src/azure-cli/azure/cli/command_modules/resource/tests/latest/recordings/test_resource_group_deployment.yaml

```diff
@@ -35,12 +35,12 @@ interactions:
       ParameterSetName:
       - --resource-group --template-file --parameters
       User-Agent:
-      - AZURECLI/2.26.0 azsdk-python-azure-mgmt-resource/18.0.0 Python/3.8.3 (Windows-10-10.0.18362-SP0)
+      - AZURECLI/2.28.0 azsdk-python-azure-mgmt-resource/19.0.0 Python/3.8.1 (Windows-10-10.0.19041-SP0)
```

Comment:
Do we really need to re-record? This code change should not change the behavior, so existing recording should work.

Reply:
Yes, it's not necessary, but better to have